### DISCUSSION TITLE
AWS Sandbox Setup

### DISCUSSION THREAD

**Discussion by Gekaus** [DefectDojo/django-DefectDojo#9130] [Q&A]
How do I setup DefectDojo in an existing AWS Sandbox without actually purchasing a subscription to the software? I.E. setting it up totally open source and by myself.

**Comment by manuel-sommer** [comment]
You can start an EC2, install docker compose and run the docker compose on the EC2. Then, you can put an ALB in front of the EC2 to easily setup HTTPS and DNS. Furthermore, you can then advance the security groups that the incomming traffic on the EC2 is only possible from the ALB and the ALB forwards directly to the right EC2 port. Pretty straight forward with Terraform.
